### PR TITLE
Add a config setting to set overlay duration

### LINKED ISSUE #11: Option to tweak duration or remove overlay
I find this package amusing but the length of time taken up by the darkened overlay has been a little frustrating at times. Would it be possible to add an option to tweak how long it is shown or remove the darkened overlay to prevent blocking the editor?

### PR DESCRIPTION
**Pull request by Haacked**
Fixes #11

### CODE REVIEW

File: lib/encourage.js [hunk 1]
```diff
@@ -52,11 +53,19 @@ export default {
...
+        console.log('EncourageAtom detected a config change in the setting `encouragementList`!');
         this.encouragements = value;
       });
     subscriptions.add(encouragementListObserver);
 
+    const encouragementDurationObserver = atom.config.observe(
+      'encourage.displayDurationMs',
+      (value) => {
+        console.log('EncourageAtom detected a config change in the setting `displayDurationMs`!');
+        this.displayDurationMs = value;
```
Comment (Haacked) [inline lib/encourage.js]: Can I assume here that `value` will be an int? I assume I don't have to put code here to deal with bad config input.

Comment (BinaryMuse) [inline lib/encourage.js]: Yeah, that's the case. The UI should reset bad values to the previous value (or a default), but even if it doesn't the value from `atom.config.get` won't be invalid (it'll be a `Number`).

Comment (Haacked) [inline lib/encourage.js]: It could be a negative number though, right? I should probably guard against that.

Comment (Haacked) [inline lib/encourage.js]: Turns out a negative number doesn't crash and just flashes the message quickly (equivalent to setting it to 0). I'm not going to bother. :smile:

**Pull request merged**